src/style.css

```diff
@@ -1,3 +1,58 @@
+/* stylelint-disable declaration-block-no-redundant-longhand-properties */
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
```

Comment:
Avoid using tag names for styling. It's better to use a class to style the 'img' elements to ensure that only the intended images are affected by these styles and to maintain CSS specificity.